### DISCUSSION TITLE
Switching to Git Discussion

### DISCUSSION THREAD

**Discussion by Alexis07Bernier** [PaulYH/GOATed_with_the_SAUCE-soen341project2023#33] [General]
We are questioning whether it is worth while switching from our discussion on discord to the git here. Let me know what you think, is this a good idea? Is it not? You tell me!

**Comment by Wandering-Fool** [comment]
I think it is quite well, because most conversation is on Discord, but it is nice to have something on the Github Page to have something that can be archive for the project.

**Reply by Alexis07Bernier** [reply]
Agreed!
